src/components/SiteHeader/SiteHeader.js

```diff
@@ -43,10 +44,10 @@ const SiteHeaderBase = kind({
 				<SiteSection className={css.frame}>
 					<Row className={css.container} align="center">
 						<Cell className={css.siteTitle} shrink>
-							<Link to="/" className={css.logo}>
-								<span className={css.image} />
+						<Link to="/" className={css.logo}>
```

Comment:
```suggestion
							<Link to="/" className={css.logo}>
```
Please keep the indent

Reply:
I fixed it.